tests/test-war-smoke/pom.xml

```diff
@@ -51,69 +52,37 @@
           <target>1.8</target>
         </configuration>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.19.1</version>
-        <configuration>
-          <excludes>
-            <exclude>**/*IntegrationTest.java</exclude>
-          </excludes>
-        </configuration>
-      </plugin>
     </plugins>
   </build>
   <profiles>
     <profile>
       <id>test.local</id>
       <activation>
         <activeByDefault>true</activeByDefault>
-        <property>
-          <name>test.mode</name>
-          <value>local</value>
-        </property>
       </activation>
       <build>
         <plugins>
           <plugin>
-            <groupId>com.spotify</groupId>
-            <artifactId>docker-maven-plugin</artifactId>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>

```

Comment:
Can the maven-failsafe-plugin config be pulled up to the parent test pom? There doesn't seem to be anything module-specific here.